scripts/travis-emscripten/emscripten.jam

```diff
@@ -0,0 +1,159 @@
+# Boost.Build support for Emscipten.
```

Comment:
Please add the source link and license into this file or create a README.md explaining it.